findmeta.py

```diff
@@ -0,0 +1,255 @@
+    images_link = items["album"]["images"]
+    max_image_res = 0
+    max_icon_size = 0
+    image_link = ""
+    icon_link = ""
+    for image in images_link:
+        if image["height"] * image["width"] > max_image_res:
+            image_link = image["url"]
+            max_image_res = image["height"] * image["width"]
+        if image["height"] < 400:
+            if image["height"] > max_icon_size:
+                max_icon_size = image["height"]
+                icon_link = image["url"]
+    track = {"Name Of Song": name,
+             "Artist(s)": artists_names,
+             "Album Artist(s)": album_artists,
+             "Album Type": album_type,
+             "Album Name": album_name,
+             "Album Release": album_release,
+             "Track Number": album_track_number,
+             "Track Duration (ms)": track_duration,
+             "Image Link": image_link,
+             "Icon Link": icon_link
+             }
+    return track
+
+
+def printDictionary(d, start_pos=0, end_pos=2):
+    """
+    Prints dictionaries in list and seperate dictionaries too
+    Since displaying all the contents at once might look cumbersome on the
+    screen, hence at a time only 4-5 tracks are displayed.
+
+    :param d: dictionary or the list of dictionaries to be printed
+              Parameters for list of dictionaries
+    :param start_pos: Position from which to start printing dictionary values
+    :param end_pos: Position upto which the dictionary is to be displayed
+
+    :returns: 1 for succesfull end of part of dictionaries being displayed
+    """
+    if type(d) is list:  # end_pos will also act as limit for no. of results
+        print("\n" + "_" * 37 + "BEGIN" + "_" * 37 + "\n")
+        for i in range(start_pos, end_pos + 1):
+            if i == len(d):
+                break
+            if len(d) != 1:  # Skip item number for single track dictionary
+                print("Item no.: {}".format(i + 1))
+            for key, value in d[i].items():
+                print("{0}: {1}".format(key, value))
+            print()
+
+        if i == len(d):
+            print("_" * 38 + "END" + "_" * 38 + "\n")
+            return 1
+        inner_choice = input("Want more results? (y/n): ")
+        if inner_choice.lower() in ['y', 'yes']:
+            printDictionary(d, start_pos=end_pos + 1, end_pos=end_pos + 5)
+
+    elif type(d) is dict:
+        print()
+        for key, value in d.items():
+            print("{0}: {1}".format(key, value))
+        print()
+
+
+if __name__ == "__main__":
+
+    while True:
+        choice = int(input("Menu:\n"
+                           "1) Search for a track\n"
+                           "2) See available music categories\n"
+                           "3) See available playlists in a category\n"
+                           "4) Get playlist tracks\n"
+                           "5) Get Track Info\n"
+                           "6) Exit\n"
+                           ))
+        print()
```

Comment:
Remove this

Reply:
I now see, there was no use of this except in the initial debugging phases.
Fixed in: https://github.com/fossdelhi/sync-music/commit/39337ba2c2c41faa71d1a46eb08a604a07052650